packages/shared/util/Recoil_RecoilEnv.js

```diff
@@ -11,16 +11,50 @@
 
 'use strict';
 
-const err = require('recoil-shared/util/Recoil_err');
+const err = require('./Recoil_err');
 
 export type RecoilEnv = {
   RECOIL_DUPLICATE_ATOM_KEY_CHECKING_ENABLED: boolean,
+  RECOIL_GKS_ENABLE: Set<string>,
 };
 
 const env: RecoilEnv = {
   RECOIL_DUPLICATE_ATOM_KEY_CHECKING_ENABLED: true,
+  RECOIL_GKS_ENABLE: new Set([
```

Comment:
I noticed this one is present tense but the other existing one is past tense. Maybe this should be `RECOIL_GKS_ENABLED` after all?

Reply:
makes sense.  consistency is good.